tests/EventListener/DeserializeListenerTest.php

```diff
@@ -61,6 +61,24 @@ public function testDoNotCallWhenRequestNotManaged()
         $listener->onKernelRequest($eventProphecy->reveal());
     }
 
+    public function testDoNotCallWhenRequestRequestIsFalse()
```

Comment:
Thinking more about it, we have `api_respond`, not `api_response`. So maybe it should be `api_receive`? @dunglas